.github/workflows/Build_all.yml

```diff
@@ -99,15 +103,21 @@ jobs:
     - name: install
       run: |
         sudo apt-get update
-        sudo apt-get install -y intel-oneapi-common-vars intel-oneapi-compiler-fortran intel-oneapi-mkl intel-oneapi-mkl-devel intel-oneapi-mpi intel-oneapi-mpi-devel
+        sudo apt-get install -y intel-oneapi-common-vars intel-oneapi-compiler-fortran intel-oneapi-mkl intel-oneapi-mkl-devel intel-oneapi-mpi intel-oneapi-mpi-devel cmake
 
     # Runs a set of commands using the runners shell
     - name: Compile
-      run: source /opt/intel/oneapi/setvars.sh && make CMP=intel BUILD=debug && make examples
+      run: |
+        source /opt/intel/oneapi/setvars.sh
+        FC=mpiifort cmake -B build -DCMAKE_BUILD_TYPE=debug
+        make -C build decomp2d
 
     # Runs a set of commands using the runners shell
     - name: Compile
-      run: source /opt/intel/oneapi/setvars.sh && make clean && make CMP=intel FFT=mkl BUILD=debug && make examples
+      run: |
+        source /opt/intel/oneapi/setvars.sh
```

Comment:
Could be best to keep the make clean here ?

Reply:
Addressed by fbc5660